net/packet/packet.go

```diff
@@ -416,13 +416,13 @@ func (q *Parsed) IsError() bool {
 			return false
 		}
 		t := ICMP4Type(q.b[q.subofs])
-		return t == ICMP4Unreachable || t == ICMP4TimeExceeded
+		return t == ICMP4Unreachable || t == ICMP4TimeExceeded || t == ICMP4ParamProblem
```

Comment:
What about ICMP4 packet too big?

Reply:
In ICMPv4 that's a code flag inside the unreachable type.

Reply:
Oh, oops.

I actually looked that up first but accidentally found an ICMPv6 doc and misread it as applying to IPv4.